src/Compilers/Test/Core/Assert/WorkItemAttribute.cs

```diff
@@ -2,9 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
```

Comment:
When the simple ctor was added, we specifically did not hide the existing one. Please revert the changes in this file.